core/readconcern/readconcern.go

```diff
@@ -40,6 +40,17 @@ func Linearizable() *ReadConcern {
 	return New(Level("linearizable"))
 }
 
+// Available specifies that the query should return data from the instance with no guarantee
+// that the data has been written to a majority of the replica set members (i.e. may be rolled back).
+func Available() *ReadConcern {
+	return New(Level("available"))
+}
+
+// Only available for operations within multi-document transactions.
+func Snapshot() *ReadConcern {
```

Comment:
Getting a vet failure here because the comment doesn't start with "Snapshot ...".

Reply:
Done.